src/views/Alteration.vue

```diff
@@ -64,27 +64,13 @@
           :disableEdit="!isRoleStaff"
         />
 
-        <!-- STAFF ONLY: Court Order and Plan of Arrangement -->
+        <!-- STAFF ONLY: Court Order/Plan of Arrangement and Staff Payment -->
         <template v-if="isRoleStaff">
-          <h2 class="mt-10">{{showTransactionalFolioNumber ? '4.' : '3.'}} Court Order and Plan of Arrangement</h2>
-          <div class="py-4">
-            If this filing is pursuant to a court order, enter the court order number. If this
-            filing is pursuant to a plan of arrangement, enter the court order number and select
-            Plan of Arrangement.
-          </div>
-
-          <div :class="{'invalid-section': invalidCourtOrder}">
-            <CourtOrderPoaShared
-              id="court-order"
-              :autoValidation="getAppValidate"
-              :draftCourtOrderNumber="getFileNumber"
-              :hasDraftPlanOfArrangement="getHasPlanOfArrangement"
-              :invalidSection="invalidCourtOrder"
-              @emitCourtNumber="setFileNumber($event)"
-              @emitPoa="setHasPlanOfArrangement($event)"
-              @emitValid="setValidCourtOrder($event)"
-            />
-          </div>
+          <CourtOrderPoa
+            class="mt-10"
+            :sectionNumber="showTransactionalFolioNumber ? '4.' : '3.'"
+            :autoValidation="getAppValidate"
+          />
```

Comment:
Here, I use the existing "local shim" component, which fits between this page and the shared component (think: layers).

As well as reusing something that was already present, this means that this page doesn't need:
- the getters for this component
- the actions for this component
- the validity function for this component